climada/engine/unsequa/calc_impact.py

```diff
@@ -261,8 +261,8 @@ def _map_impact_calc(self, sample_iterrows):
         haz = self.haz_input_var.evaluate(**haz_samples)
 
         imp = Impact()
-
-        imp.calc(exposures=exp, impact_funcs=impf, hazard=haz)
+        exp.assign_centroids(haz, overwrite=False)
+        imp.calc(exposures=exp, impact_funcs=impf, hazard=haz, assign_centroids=False)
```

Comment:
This is now a bit awkward, but it probably works fine. We should check whether the computation time remains stable. Have you already checked that on the test code? Shall I do some more testing?

I think the same change is needed for `unsequa.calc_cost_benefit` right?

Reply:
Yes, same, same for `calc_cost_benefit`. I just checked it in.

And yes, I have checked the computational impact, not by computation time itself which is marginal for the test cases I can get my hands on, but by number of calls to `assign_centroids` with `overwrite=True`.

And no, thanks, that should be fine. You don't have to do more testing. Unless you object I'm going to merge.